app/views/core/RootView.coffee

```diff
@@ -116,6 +120,15 @@ module.exports = class RootView extends CocoView
     #location.hash = hash
     @renderScrollbar()
 
+  render: ->
+    try
+      @initializeMetaBinding()
```

Comment:
Item for discussion:

I initially had this running in afterRender to avoid touching the render process.  However this resulted in a significant delay in updating the page title for pages that took longer to render.  I've added it here and surrounded in a try catch to make sure we do not break the normal render flow.

Any other ways we could do this.  Only other thought I had was to add a `beforeRender` callback but that would also be touching the render process and I figured this was simpler.

Reply:
This seems fine, agree with not going with `beforeRender` option.